### PR TITLE
feat: add accountapp login synthetic monitor

### PR DESCRIPTION
**Pull request by lemeurherve**
To monitor accountapp and its correct connection with LDAP, I've manually created the `datadog_monitoring` user on accounts.jenkins.io (with jenkins-infra-team@googlegroups.com as email), then with this PR a synthetic test is simulating a connection to accounts.jenkins.io to check if everything is OK.

Ref: https://github.com/jenkins-infra/helpdesk/issues/1323

### CODE REVIEW

File: synthetics_accountApp.tf [hunk 1]
```diff
@@ -13,7 +13,7 @@ resource "datadog_synthetics_test" "accountsApp" {
   options_list {
     tick_every = 900
   }
-  name    = "account.jenkins.io"
+  name    = "accounts.jenkins.io"
```
Comment (lemeurherve) [inline synthetics_accountApp.tf:16]: I've kept this previous monitor but we might want to remove it.

**Comment by lemeurherve** [comment]
In draft as the new credential for "datadog_monitor"'s Jenkins user password isn't stored in the secrets yet.

File: synthetics_accountApp.tf [hunk 2]
```diff
@@ -1,24 +0,0 @@
...
-  }
-  name    = "account.jenkins.io"
-  message = "Notify @pagerduty"
-  tags = [
-    "jenkins.io",
-    "production"
-  ]
-
-  status = "live"
-}
```
Comment (dduportal) [inline synthetics_accountApp.tf:24]: I'm wondering if we should keep it or not.

If the new monitors fails, then it means that either the acount application is down (Kubernetes down/DNS/ no LB/App failing/OOM/etc.) and / OR LDAP is down.

I wonder if we shouldn't keep both to ensure that we can quickly, just by staring at the monitors, where is the "area of failure"

WDYT?

Comment (dduportal) [reply synthetics_accountApp.tf]: (sorry for the additional work, I did not thought about it on first review)

**Review by dduportal — APPROVED** [review]
👏

**Comment by lemeurherve** [comment]
Not sure why terraform can't find `datadog_jenkinsuser_password` while there is a `TF_VAR_datadog_jenkinsuser_password` declared in the Jenkinsfile_k8s though

**Pull request merged**